### PR TITLE
Update mac os development guide

### PR DESCRIPTION
**Pull request by grimadas**
Fix max os development guide that work now with `brew`

### CODE REVIEW

**Comment by devos50** [comment]
ok to test

**Comment by devos50** [comment]
add to whitelist

**Comment by cclauss** [comment]
Why can't we just have a __requirements.txt__ file for Python dependencies like almost every other project?  Or use pipenv or pipx?

__pip install -r requirements.txt__ works on Mac, Linux, and Windows.

Installing things like __py27-Pillow py27-twisted py27-cherrypy3 py27-cffi py27-chardet py27-configobj py27-gmpy2 py27-pycparser py27-numpy py27-idna py27-cryptography py27-decorator py27-netifaces py27-service_identity py27-asn1-modules py27-pyinstaller py27-pyqt5 py27-sqlite py27-matplotlib py27-libnacl__ is guaranteed to install unpeggable (and usually out-of-date) modules.

File: doc/development/development_on_osx.rst [hunk 1]
```diff
@@ -89,16 +89,17 @@ Other Packages
...
 
-.. code-block:: none
+.. code-block:: bash
 
     brew install homebrew/python/pillow gmp mpfr libmpc libsodium
     sudo easy_install pip
     pip install --user cython  # Needs to be installed first for meliae
-    pip install --user cherrypy cffi chardet configobj cryptography decorator gmpy2 idna meliae netifaces numpy pillow psutil pyasn1 pycparser scipy twisted service_identity libnacl bitcoinlib
+    pip install --user cherrypy cffi chardet configobj cryptography decorator gmpy2 idna meliae netifaces numpy pillow \
+    psutil pyasn1 pycparser scipy lz4 pyopenssl Twisted==16.4.1 networkx service_identity libnacl bitcoinlib
```
Comment (cclauss) [inline doc/development/development_on_osx.rst]: Can we put this into a __requirements.txt__ file (maybe with pegged versions?) and advocate __pip install -r requirements.txt__ instead?

**Comment by devos50** [comment]
retest this please

**Pull request merged**